Application/EdFi.Ods.AdminApi/Infrastructure/Extensions/AdminModelExtensions.cs

```diff
@@ -39,4 +39,9 @@ public static IList<Profile> Profiles(this Application application)
     {
         return application?.Vendor?.VendorId;
     }
+
+    public static int? EducationOrganizationId(this Application application)
+    {
+        return application?.ApplicationEducationOrganizations?.FirstOrDefault()?.EducationOrganizationId;
+    }
```

Comment:
Overall this looks like a good temporary solution. Long term, we need to ask ourselves why are we using `FirstOrDefault`? Why aren't we returning _all_ education organizations? I'm going to make note of this [in a page](https://techdocs.ed-fi.org/pages/viewpage.action?spaceKey=EFTD&title=Admin+API+2.0+Interface) that I started for documenting proposed interface changes in Admin API 2.0.